tests/test_views.py

```diff
@@ -9,6 +9,14 @@
 from fobi_custom.plugins.form_elements.fields import types as fobi_types
 
 
+@pytest.mark.django_db
+def test_study_area_create():
```

Comment:
We'll want to switch this to something like:

```suggestion
def test_home_page_loads():
```

Reply:
Thx 😅